res/css/views/messages/_SenderProfile.scss

```diff
@@ -13,3 +13,41 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
+.mx_SenderProfile_name {
+    font-weight: 600;
+}
+
+.mx_SenderProfile_color1 {
+    color: #1e7ddc;
+}
+
+.mx_SenderProfile_color2 {
+    color: #a756a8;
+}
+
+.mx_SenderProfile_color3 {
+    color: #7ac9a1;
+}
+
+.mx_SenderProfile_color4 {
+    color: #f2809d;
+}
+
+.mx_SenderProfile_color5 {
+    color: #ffc666;
+}
+
+.mx_SenderProfile_color6 {
+    color: #76ddd7;
+}
+
+.mx_SenderProfile_color7 {
+    color: #45529b;
+}
+
+.mx_SenderProfile_color8 {
+    color: #bfd251;
+}
```

Comment:
Should these be constants in the theme css?

Reply:
Yeah, good point